src/main/java/org/kainos/ea/db/AuthDao.java

```diff
@@ -0,0 +1,66 @@
+package org.kainos.ea.db;
+
+import org.apache.commons.lang3.time.DateUtils;
+import org.kainos.ea.cli.Login;
+import org.kainos.ea.cli.Role;
+
+import java.sql.*;
```

Comment:
IntelliJ will have combined this into a wildcard import, can you make it not a wildcard import